docs/docs/labelling-issues.md

```diff
@@ -0,0 +1,99 @@
+
+## How to label an issue
+
+Ideally every issue should have a single `🏷 type:` label applied to it. Optionally a `status: ` label, or other labels may also be applied.
+
+Before continuing, get familiar with [Gatsby's issue labels and their descriptions](https://github.com/gatsbyjs/gatsby/issues/labels).
+
+The broad steps to labelling an issue are:
+
+- Read an issue
+- Choose the labels that apply to that issue
+- That's it
+
+The rest of this document will describe how to choose the right labels for an issue.
+
+### Find an issue that you're interested in
+
+Start with [Gatsby's issues list](https://github.com/gatsbyjs/gatsby/issues) and scroll through until you see a recent one that strikes your interest. Alternatively you can view the [list of unlabelled issues](https://github.com/gatsbyjs/gatsby/issues?q=is%3Aopen+is%3Aissue+no%3Alabel).
+
+### Read the issue
+
+Read the issue and any comments to understand what the issue is about.
+
+### Choose one `type: ` label
+
+Choose a type label from the _labels_ dropdown to the right-hand side of the issue.
+
+!(GitHub label dropdown)(./images/github-label-list.png)
+
+You can check through the [label descriptions](https://github.com/gatsbyjs/gatsby/issues/labels) for more information on each one.
+
+The most common type of issue is `type: question or discussion`, typically you can apply this to issues that are open-ended or have no clear next step.
+
+It's OK to change the type of an issue as more information becomes available. What starts as `type: question or discussion`, might later need to be changed to `type: bug`.
+
+Changing labels is quick and easily reversible, so don't worry too much about applying a "wrong" label.
+
+Choose an appropriate `type: ` label and you're ready to move on to the next step.
+
+### Choose a `status: ` label (optional)
+
+Check through the [`📍 status: ` labels (and their descriptions)](https://github.com/gatsbyjs/gatsby/issues/labels), if any apply to this issue add them as necessary.
+
+Examples of applying `📍 status: ` labels might be:
+
+- An issue that depends on an external dependency being changed could be labelled with `📍 status: blocked`
+
+- An issue with a clear description of how it can be resolved could be labelled `📍 status: help wanted`.
+
+- An issue that's missing information required to help the author could be labelled with `📍 status: needs more info`
+
+- An issue describing a bug without clear steps to reproduce could be labelled with `📍 status: needs reproduction`
+
+- An issue describing a bug where there's steps to reproduce the bug _and_ you (or someone else) have confirmed that they can reproduce the bug can be labelled `📍 status: confirmed`
+
+### TODO - is there a way to publicly list these people?  https://github.com/orgs/gatsbyjs/teams/admin/members
+- An issue describing a feature can be labelled `📍 status: confirmed` if a member of Gatsby's Admin team has approved the feature
```

Comment:
Not that I've seen, unfortunately. Teams are pretty much invisible unless you're part of them.

Reply:
Hrmm ok, I'll try and reword that somehow... "Don't do this unless you know you're allowed to" isn't too helpful though! Maybe we can drop this line.